Makefile

```diff
@@ -3,32 +3,28 @@ GHCS = 7.8.4 7.10.3 8.0.2 8.2.2
 .PHONY: all
 all: build test package doc tags
 
-default.nix: extensible-effects.cabal
-	cabal2nix ./. > ./default.nix
-
 .PHONY: init
 init:
-	cabal sandbox init
-	cabal clean
+	stack init
+	stack clean
 
 .PHONY: build
 build: init
-	cabal install --only-dependencies --enable-tests --enable-benchmarks
-	cabal configure -flib-Werror --enable-tests --enable-benchmarks -v2 -O2
```

Comment:
@power-fungus what's the equivalent of `cabal configure` for `stack`?

Reply:
@suhailshergill `stack build` does configure everything depending on your provided flags. Also, package flags can be set in the yaml file (in this case `lib-Werror`) and in the cabal file, like `-O2`...